### PR TITLE
User ID Clarification

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3042

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by shivtorov** [comment]
asterisk character (&ast;) in the source text needs to be replaced with `&ast;`

**Comment by scanum** [comment]
@TokyoScarab Can these changes be made? Thanks!

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/configure-cloud-connector-integration-with-your-office-365-tenant.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ After adding your users to Office 365, enable their accounts for Phone System in
   ```
 
     > [!NOTE]
-    > You can also specify a user's Identity by their SIP address, User Principal name (UPN), domain name and username (domain\username), and display name in Active Directory ("Bob Kelly"). 
+    > A user identity can be specified using the user's SIP address, user principal name (UPN), or the user's Active Directory display name (for example, "Bob Kelly"). The asterisk () wildcard character can also be used with the Display Name as the user Identity. For example, the Identity " Smith" returns all the users who have a display name that ends with the string value " Smith".
```
Comment (TokyoScarab) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/configure-cloud-connector-integration-with-your-office-365-tenant.md]:
```suggestion
    > A user identity can be specified using the user's SIP address, user principal name (UPN), or the user's Active Directory display name (for example, "Bob Kelly"). The asterisk (&ast;) character can also be used with the Display Name as the user Identity. For example, the Identity "Smith" returns all the users who have a display name that ends with the string value " Smith".
```

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/configure-cloud-connector-integration-with-your-office-365-tenant.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ After adding your users to Office 365, enable their accounts for Phone System in
   ```
 
     > [!NOTE]
-    > You can also specify a user's Identity by their SIP address, User Principal name (UPN), domain name and username (domain\username), and display name in Active Directory ("Bob Kelly"). 
+    > A user identity can be specified using the user's SIP address, user principal name (UPN), or the user's Active Directory display name (for example, "Bob Kelly"). The asterisk (&ast;) character can also be used with the Display Name as the user Identity. For example, the Identity "Smith" returns all the users who have a display name that ends with the string value " Smith".
```
Comment (TokyoScarab) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/configure-cloud-connector-integration-with-your-office-365-tenant.md]:
```suggestion
    > A user identity can be specified using the user's SIP address, user principal name (UPN), or the user's Active Directory display name (for example, "Bob Kelly"). The asterisk (&ast;) character can also be used with the Display Name as the user Identity. For example, the Identity "* Smith" returns all the users who have a display name that ends with the string value " Smith".
```

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/configure-cloud-connector-integration-with-your-office-365-tenant.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ After adding your users to Office 365, enable their accounts for Phone System in
   ```
 
     > [!NOTE]
-    > You can also specify a user's Identity by their SIP address, User Principal name (UPN), domain name and username (domain\username), and display name in Active Directory ("Bob Kelly"). 
+    > A user identity can be specified using the user's SIP address, user principal name (UPN), or the user's Active Directory display name (for example, "Bob Kelly"). The asterisk (&ast;) character can also be used with the Display Name as the user Identity. For example, the Identity "* Smith" returns all the users who have a display name that ends with the string value " Smith".
```
Comment (nenonix) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/configure-cloud-connector-integration-with-your-office-365-tenant.md]:
```suggestion
    > A user identity can be specified using the user's SIP address, user principal name (UPN), or the user's Active Directory display name (for example, "Bob Kelly"). The asterisk (\*) character can also be used with the Display Name as the user Identity. For example, the Identity "\*Smith" returns all the users who have a display name that ends with the string value "Smith".
```

**Comment by TokyoScarab** [comment]
Thanks! I'll keep that in mind! :)

**Pull request merged**